pkg/pulumiyaml/run_test.go

```diff
@@ -80,6 +81,10 @@ func (m MockPackage) ResolveFunction(typeName string) (FunctionTypeToken, error)
 	return FunctionTypeToken(typeName), nil
 }
 
+func (m MockPackage) IsResourcePropertySecret(typeName ResourceTypeToken, propertyName string) (bool, error) {
+	return m.isResourcePropertySecret(typeName.String(), propertyName)
```

Comment:
There's a test currently failing here, we need to either add an implementation of  `isResourcePropertySecret` to https://github.com/pulumi/pulumi-yaml/blob/0228354d1e833d8cd666d7957e60413a6563061f/pkg/pulumiyaml/run_test.go#L168 or  do something like we do for ResolveFunction/ResolveResource where we return `false, nil` by default here.